NOTICE.txt

```diff
@@ -7,41 +7,6 @@ NOTICES:
 This document includes a list of open source components used in Mattermost Mobile, including those that have been modified.
 
 --------
-## @babel/runtime
```

Comment:
this is still being used

Reply:
We don't list devDependencies in the NOTICE.txt.